tests/plugins/dialog/manual/pagescroll.html

```diff
@@ -0,0 +1,8 @@
+<div style="min-height: 100vh"></div>
+<textarea id="editor"></textarea>
+<script>
+	if ( CKEDITOR.env.ie || bender.tools.env.mobile ) {
```

Comment:
Is there any other reason to ignore manual tests in IEs than lack of support for `vh` unit? If no, then this issue could be fixed with using large `px` value for `height`.

Reply:
Still ignored…